docs/src/rsh/compute/index.md

```diff
@@ -1641,20 +1641,32 @@ lastConsensusTime()
 The @{defn("lastConsensusTime")} primitive returns the network time of the last publication of the DApp.
 This may not be available if there was no such previous publication, such as at the beginning of an application before the first publication.
 
-:::note
-Why is there no `thisConsensusTime`?
-Some networks do not support observing the time of a consensus operation until after it has finalized.
-This aides scalability, because it increases the number of times when an operation could be finalized.
```

Comment:
We might want to retain this info in the form of cautionary words on the docs for `thisConsensusTime`.

A more extreme opinion might be that we name it `unsafeGetCurrentConsensusTime()` or something silly like that.